login-module/webpack.config.cjs

```diff
@@ -1,9 +1,12 @@
-const path = require('path');
+const path = require('path')
 
 module.exports = {
 	entry: './src/index.js',
 	output: {
+		libraryExport: 'default',
+		libraryTarget: 'umd',
+		library: 'GraphAcademyLogin',
```

Comment:
The module will be available on the global scope with the name `GraphAcademyLogin`. Usage in the browser:

```js
new GraphAcademyLogin({})
```